ProjectVoyage/View Controllers/Markdown/MarkdownExportVC.h

```diff
@@ -0,0 +1,19 @@
+//
+//  MarkdownExportVC.h
+//  ProjectVoyage
+//
+//  Created by Gui David on 7/15/22.
+//
+
+#import <UIKit/UIKit.h>
+#import "Chat.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MarkdownExportVC : UIViewController
+
+@property (strong, nonatomic) Chat *chat;
```

Comment:
nit: objc etiquette is to do: 
```suggestion
@property (nonatomic, strong) Chat *chat;
```

Reply:
Changed it! Xcode unfortunately automatically shows up in that form (strong, nonatomic) when you CTRL-drag the object. I will be more mindful to swap them next time